src/app/content/studyGuides/hooks/openStudyGuides.ts

```diff
@@ -0,0 +1,40 @@
+import { GetHighlightsColorsEnum, GetHighlightsSetsEnum } from '@openstax/highlighter/dist/api';
+import { ActionHookBody } from '../../../types';
+import { actionHook } from '../../../utils';
+import { summaryPageSize } from '../../constants';
+import { book as bookSelector, highlightLocationFilters } from '../../selectors';
+import { formatReceivedHighlights, loadUntilPageSize } from '../../utils/highlightLoadingUtils';
+import { openStudyGuides, receiveSummaryStudyGuides } from '../actions';
+import { allColors } from '../constants';
+import * as select from '../selectors';
+
+export const hookBody: ActionHookBody<typeof openStudyGuides> = ({
+  dispatch,
+  getState,
+  highlightClient,
+}) => async() => {
+  const state = getState();
+  const book = bookSelector(state);
+  const totalCounts = select.totalCountsPerPageOrEmpty(state);
+  const locationFilters = highlightLocationFilters(state);
+
+  const summaryNeedsInitialization = () => select.summaryStudyGuidesHighlights(state) === null
+    && select.summaryIsLoading(state) === false;
+
+  if (summaryNeedsInitialization()) {
```

Comment:
here we can use the same loadMore function as the loadMore hook right?

Reply:
I think it should actually dispatch for loadMore so it also sets the loading state